src/CommunityToolkit.Maui.MediaElement/MediaElement.shared.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace CommunityToolkit.Maui.MediaElement;
+
+public partial class MediaElement : View, IMediaElement
+{
+	public MediaElement()
+	{
+	}
+
+	public MediaSource? Source { get; set; }
+
+	public TimeSpan Position { get; set; }
```

Comment:
I am not convinced that the base class "MediaElement" should provide properties bond to files such as Position. When playing a stream for instance, this property is kinda useless. Same might go for duration when there is a live stream which does not have a duration. I'd vote for seperating these into different abstractions or at least interface implementations.

Reply:
Thanks for the feedback! The API is still in flux, it's marked as a draft still :)

Reply:
Also, I'm not really sure if that's true. If you look at a live stream on YouTube, I can set the position back in time to still watch from that point on, so it might be useful for that just as well. Duration might not be, this was also already pointed out above.